go/runtime/committee/committee.go

```diff
@@ -86,14 +97,35 @@ Members:
 			}
 		}
 
-		_, err := cw.nw.WatchNode(ctx, member.PublicKey)
-		if err != nil {
+		filtered = append(filtered, member)
+	}
+
+	// If the set of (filtered) committee members did not change, there is no need to trigger a
+	// reset and recreate everything. Nodes will be updated anyway.
+	var cid hash.Hash
+	cid.From(filtered)
```

Comment:
should `filtered` be sorted so that order won't affect the hash? since looking at election code i think it's possible that the order changed but actual nodes are the same: https://github.com/oasislabs/oasis-core/blob/9e521ce7c57ca0e3cab7178eb43f5bca55b532e1/go/consensus/tendermint/apps/scheduler/scheduler.go#L477-L497

Reply:
Yeah it should be, thanks!

Reply:
Done.

Reply:
:heavy_check_mark: